integration/portal/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/main.ts',
```

Comment:
Why are we using webpack directly instead of angular cli? Wouldn't using angular cli be a more accurate representation of how people generally use single-spa-angular?

^ I say this as someone who likes using webpack directly and finds angular schematics and angular builder confusing. But if we're going for e2e tests, I would have expected angular cli since that's what most people use

Reply:
The `portal` is not an angular app. It's a single web-app that uses `single-spa` and gets bundled and served via webpack.

Reply:
Ah I see - I didn't realize that. Yes in that case it makes a ton of sense to do it this way